components/developer/gdb/Makefile

```diff
@@ -20,30 +20,68 @@
 #
 
 #
-# Copyright (c) 2011, 2014, Oracle and/or its affiliates. All rights reserved.
-# Copyright 2017 Gary Mills
+# Copyright (c) 2011, 2017, Oracle and/or its affiliates. All rights reserved.
+# Copyright 2017, 2021 Gary Mills
 # Copyright 2019 Andreas Wacknitz
 #
 
 BUILD_BITS= 64
+# There are a large number of macros in the GCB code written
+# specifically for GCC's preprocessor. Studio does not expand
+# these macros the same way GCC does, and the resulting gdb is
+# not usable. Building gdb with a GCC >= 4.7.2 produces very
+# good results, even on SPARC.
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		gdb
-COMPONENT_VERSION=	7.10.1
-COMPONENT_REVISION=	6
+COMPONENT_VERSION=	8.0
+# COMPONENT_REVISION=	0
 COMPONENT_SUMMARY=	GDB: The GNU Project Debugger
 COMPONENT_CLASSIFICATION=Development/System
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
 COMPONENT_ARCHIVE_HASH=	\
-     sha256:ff14f8050e6484508c73cbfa63731e57901478490ca1672dc0b5e2b03f6af622
+     sha256:8968a19e14e176ee026f0ca777657c43456514ad41bb2bc7273e8c4219555ac9
 COMPONENT_ARCHIVE_URL=	https://ftp.gnu.org/gnu/gdb/$(COMPONENT_ARCHIVE)
 COMPONENT_PROJECT_URL=	https://www.gnu.org/software/gdb/
 
 CONFIGURE_DEFAULT_DIRS=no
 
+# Set python version used by this product
+PYTHON_VERSION=3.7
+
 include $(WS_MAKE_RULES)/common.mk
 
+# The traditional output from gmake test omits some
+# test results information that is contained in the 
+# build/<arch>/gdb/testsuite/gdb.sum file, so use that file instead
+# for test results comparison.
+COMPONENT_POST_TEST_ACTION += \
+	( $(MV) $(COMPONENT_TEST_OUTPUT) $(COMPONENT_TEST_OUTPUT)-orig ; \
+	$(CP) $(BUILD_DIR_64)/gdb/testsuite/gdb.sum $(COMPONENT_TEST_OUTPUT) )
+
+# Results output is different on Intel vs. SPARC.
+# Note that comparisons against master results 
+# are not clean since results will still vary.
+COMPONENT_TEST_MASTER = \
+        $(COMPONENT_TEST_RESULTS_DIR)/results-$(MACH)-$(BITS).master
+
```

Comment:
You have the test machinery in the Makefile but your PR lacks the corresponding test/results-i386-64.master file.
You can simply create the test folder and "cp build/test/amd64/results-64.snapshot test/results-i386-64.master" after you run "gmake test".